src/main/kotlin/com/sainsburys/k2zpl/builder/ZplBuilder.kt

```diff
@@ -2,6 +2,7 @@
 
 package com.sainsburys.k2zpl.builder
 
+import LazyCommand
```

Comment:
I'm struck how here we're able to import it was `import LazyCommand`, but with other types we need to use `import com.sainsburys.kzpl...`.

And indeed, I see the `package` declaration is missing in `LazyCommand.kt`! 🙈